### PR TITLE
Feature/matsumoto yuikei

### PR DESCRIPTION
**Pull request by YuikeiMatsumoto**
Please check my branch.

### CODE REVIEW

**Review by BeerShigachi — CHANGES_REQUESTED** [review]
Please remove irrelevant files. The change does not meet the requirements.

File: README.md
Comment (BeerShigachi) [inline README.md:1]: Do not include this file.

File: ayato.txt
Comment (BeerShigachi) [inline ayato.txt:1]: Do not include this file.

File: dummy.txt
Comment (BeerShigachi) [inline dummy.txt:1]: Do not include this file.

File: lam.txt
Comment (BeerShigachi) [inline lam.txt:1]: Do not include this file.

File: riku.txt
Comment (BeerShigachi) [inline riku.txt:1]: Do not include this file.

File: ryuta.txt
Comment (BeerShigachi) [inline ryuta.txt:1]: Do not include this file.

File: sample.txt
Comment (BeerShigachi) [inline sample.txt:1]: Do not include this file.

**Review by BeerShigachi — CHANGES_REQUESTED** [review]

**Comment by YuikeiMatsumoto** [comment]
I've corrected my branch. Please check it again.

**Review by BeerShigachi — CHANGES_REQUESTED** [review]
Do not delete README.md

**Comment by YuikeiMatsumoto** [comment]
I've resolved the conflict of my branch. Please check it.

**Pull request closed without merging**